### PR TITLE
Cleanup test cases

### PR DESCRIPTION
**Pull request by IchHabRecht**

### CODE REVIEW

File: src/TestingFramework/TestCase/FunctionalTestCase.php [hunk 1]
```diff
@@ -157,52 +144,50 @@
...
+     * @return void
      */
-    protected function getInstanceIdentifier()
+    protected function setUp()
     {
-        return FunctionalTestCaseBootstrapUtility::getInstanceIdentifier(get_class($this));
+        if (!defined('ORIGINAL_ROOT')) {
+            $this->markTestSkipped('Functional tests must be called through phpunit on CLI');
+        }
+        $this->testSystem = TestSystemFactory::getInstance(get_class($this));
```
Comment (helhum) [inline src/TestingFramework/TestCase/FunctionalTestCase.php]: I'd prefer `TestSystemFactory::createFromTestClassName`

**Review by helhum — APPROVED** [review]
very nice cleanup. only have a minor comment

File: src/TestingFramework/TestCase/FunctionalTestCase.php (hunk 1 above)
Comment (IchHabRecht) [reply src/TestingFramework/TestCase/FunctionalTestCase.php]: Hm, but wouldn't that mean that there is a dependency between the class name and the testSystem instance?

Comment (IchHabRecht) [reply src/TestingFramework/TestCase/FunctionalTestCase.php]: Switched method name to getInstanceByIdentifier

**Pull request merged**